src/main/java/frc/robot/commands/OneBar/ArmToHigh.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.OneBar;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.Wrist.*;
+import frc.robot.subsystems.OnebarSubsystem;
+import frc.robot.subsystems.WristSubsystem;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class ArmToHigh extends SequentialCommandGroup {
```

Comment:
Does this need to be a sequential command group? The two commands are using two different subsystems. How about we use ParallelCommandGroup instead?

(Same comment for other commands)